packages/components/organisms/f-offers/src/components/Results.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+    <content-cards
+        v-if="canShowContentCards"
+        #default="{ cards }"
+        :user-id="globalUserId"
+        :api-key="brazeApiKey"
+        :locale="$i18n.locale"
+        :push-to-data-layer="pushToDataLayer"
+        @has-loaded="hasLoaded"
+        @get-card-count="cardsReceived"
+        @on-error="onError"
+        @voucherCodeClick="openModal"
+    >
```

Comment:
Tiny: can you please move closing brackets in this file to the last prop for consistency?
```
...
@voucherCodeClick="openModal">
etc
```